media-push/develop/integration-best-practices.mdx

```diff
@@ -73,105 +71,42 @@ the following two ways to implement it:
     whether the priority function is supported and what the name of the
     priority field is.
 
-### Switch domain name
-
-Take the following steps to switch a domain name:
-
-1. Set the primary domain name based on the location of your service
-server:
--   If the DNS address of the service server is in mainland China, set
-    the primary domain name to `api.sd-rtn.com`.
--   If the DNS address of the service server is located in a country or
-    region other than mainland China, set the primary domain name to
-    `api.agora.io`.
-2. If the primary domain name fails to initiate a RESTful API request,
-use the primary domain name to retry.
-
-3. If the retry in step 2 still fails, use the alternate domain
-name to retry.
-
--   If the current primary domain name is `api.sd-rtn.com`, the alternate
-    domain name is `api.agora.io`.
--   If the current primary domain name is `api.agora.io`, the alternate
-    domain name is `api.sd-rtn.com`.
-
-4. If the retry in step 3 still fails, use the domain name adjacent to
-the current region to retry.
-
-For example, suppose your business server is located in Europe. You set the primary domain name to `api.agora.io`, and
-the business server resolves the primary domain name to Germany. Germany is located in Central Europe (`api-eu-central-1.agora.io`).
-Check the domain name table below - the adjacent area is West Europe (`api-eu-west-1.agora.io` or `api-eu-west-1.sd-rtn.com`).
+For use cases with higher quality requirements, requesting multiple Media push tasks with different tagging enables isolated resources for these tasks and raises the success rate. Media push supports room-level isolation for tasks. For example, if there is a task with `"name":"my-task"` in the original request, you can add a suffix to the `name` parameter to realize isolation at the computer room level. Use `"name":"my-task-agora-isolate-1"` and `"name":"my-task-agora-isolate-2"` to start additional tasks. Note that:
```

Comment:
media push tasks but Media Push ( to be consistent)

Reply:
Changed all occurrences to Media Push